src/AppBundle/Resources/views/Admin/Client/Report/checklist.html.twig

```diff
@@ -62,6 +62,34 @@
         </p>
     {% endif %}
 
+    {% if report.deputydocuments %}
```

Comment:
I'd name this getters deputyDocuments, more in line with the naming we use. Twig is case insensitive, but  we should respect the case as much as possible to make it easier to find&replace when needed